From 54e9d1a87ea67bd2ae5933bd9c1542cb2803bcf8 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 4 Apr 2011 22:46:29 -0500
Subject: Look for NEWS in order to find etc/ rather than GNU.

Emacs will now look for NEWS in order to find etc/ rather than GNU.

Since Debian doesn't include GNU in the Debian packages (the common
licenses are in /usr/share/common-licenses), Emacs will now look for
NEWS to find the etc directory instead of GNU.

Provided-by: Sven Joachim <svenjoac@gmx.de>
Originally-reported-by: Bernhard Michler <Boregard@gmx.net>
Date: Mon, 28 Apr 2008 11:20:23 +0200
Added-by: Rob Browning <rlb@defaultvalue.org>
Status: Debian specific
---
 src/callproc.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/callproc.c b/src/callproc.c
index 6898bd9..84b1c30 100644
--- a/src/callproc.c
+++ b/src/callproc.c
@@ -1509,14 +1509,14 @@ init_callproc ()
 
       srcdir = Fexpand_file_name (build_string ("../src/"),
 				  build_string (PATH_DUMPLOADSEARCH));
-      tem = Fexpand_file_name (build_string ("GNU"), Vdata_directory);
+      tem = Fexpand_file_name (build_string ("NEWS"), Vdata_directory);
       tem1 = Ffile_exists_p (tem);
       if (!NILP (Fequal (srcdir, Vinvocation_directory)) || NILP (tem1))
 	{
 	  Lisp_Object newdir;
 	  newdir = Fexpand_file_name (build_string ("../etc/"),
 				      build_string (PATH_DUMPLOADSEARCH));
-	  tem = Fexpand_file_name (build_string ("GNU"), newdir);
+	  tem = Fexpand_file_name (build_string ("NEWS"), newdir);
 	  tem1 = Ffile_exists_p (tem);
 	  if (!NILP (tem1))
 	    Vdata_directory = newdir;
